extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -216,22 +211,25 @@ function Autoplay({
           onClick={onOverlayClick}
         ></div>
       )}
-
-      {/* TODO(wg-bento): Global styling.
-          https://github.com/ampproject/wg-bento/issues/7 */}
-      <style>{autoplayCss}</style>
     </>
   );
 }
 
 /**
  * @return {!PreactDef.Renderable}
  */
-const AutoplayIconContent = once(() =>
-  [1, 2, 3, 4].map((i) => (
-    <div className="amp-video-eq-col" key={i}>
-      <div className={`amp-video-eq-filler amp-video-eq-${i}-1`}></div>
-      <div className={`amp-video-eq-filler amp-video-eq-${i}-2`}></div>
+const AutoplayIconContent = once(() => {
+  const classes = useStyles();
+  return [1, 2, 3, 4].map((i) => (
+    <div className={`${classes.eqCol} amp-video-eq-col`} key={i}>
+      <div
+        style={{animationName: classes['amp-video-eq-animation']}}
+        className={`amp-video-eq-filler ${classes[`eq-${i}-1`]}`}
+      ></div>
+      <div
+        style={{animationName: classes['amp-video-eq-animation']}}
+        className={`amp-video-eq-filler ${classes[`eq-${i}-2`]}`}
+      ></div>
```

Comment:
@samouri Do I need to unwrap this loop since the member access for `classes` is dynamic?